test/lit.cfg

```diff
@@ -346,6 +346,35 @@ else:
 config.substitutions.append( ('%swift-tensorflow-extra-options', swift_tensorflow_extra_options) )
 config.substitutions.append( ('%swift-tensorflow-test-run-extra-options', swift_tensorflow_test_run_extra_options) )
 
+# Discover python lib files for dynamic loading when testing Python Interoperability
+def get_python_library_file_path_for(major_version):
+    current_major_version = sys.version_info.major
+    if current_major_version == major_version:
+        return os.path.join(sys.prefix, 'Python')
+    exec_name = ''
+    if major_version == 2:
+        exec_name = 'python'
+    elif major_version == 3:
+        exec_name = 'python3'
+    else:
+        raise RuntimeError('Python version {}.x is not supported'.format(major_version))
+
+    try:
+        library_file_path = subprocess.check_output(
+            [exec_name, '-c', 
```

Comment:
I'm not sure that `subprocess.check_output([exec_name, ...])` is a robust way to check that the Python 2/3 libraries are installed. Ideally, the logic should be consistent with the library searching logic in [`PythonLibrary.loadPythonLibrary()`](https://github.com/apple/swift/blob/e7f674e7a664fa2d35c1f5323a304e57e8b8cfd1/stdlib/public/Python/PythonLibrary.swift#L183).

Other reviewers: any ideas?

Reply:
It isn't quite as clean in the test results, but I think you can use: PythonLibrary() in stdlib/public/Python. Pass in an explicit version, and then short-circuit all the tests if that version of python is not loadable. You have to be very careful to do this as the first thing of the program. In this model, you would blindly run the tests for all versions, but only the versions that exist would run. Thus, lit would not need to be aware of this.